routes/channel.js

```diff
@@ -28,7 +28,21 @@ router.post(
 	channelLoad,
 	postValidatorMessages
 )
-router.post('/:id/events', authRequired, channelIfActive, postEvents)
+router.post(
+	'/:id/events',
+	authRequired,
+	celebrate({ body: schema.events }),
+	channelIfActive,
+	postEvents
+)
+router.post(
```

Comment:
why is this not a part of the /events post; there's a `type` in each event

you can just make it so that if there's an event with a `type` == "CLOSE", it should require authentication to be from channel.creator

Reply:
This is because it requires `channelLoad`. Which was removed earlier for performance

Reply:
there's a further optimization we can do... add `channelIfCreator` in the middlewares, which will only allow requests from the channel creator